rmr/utils/datetime.py

```diff
@@ -12,3 +13,70 @@ def strptime(date_string, format, default_tz=timezone.utc):
     if timezone.is_naive(dt):
         dt = timezone.make_aware(dt, timezone=default_tz)
     return timezone.localtime(dt)
+
+
+NOW = object()
+
+
+def get_date_range(
+        start_date: int = None,
+        end_date: int = NOW,
+        default_range: int = None,
+        max_range: int = None,
+) -> tuple:
+    """
+    get datetime ranges
+
+    :return: range from datetime to datetime
+    :rtype: (datetime, datetime)
+    """
+
+    if end_date is NOW:
+        end = timezone.now()
+    elif end_date is None:
+        end = None
+    else:
+        try:
+            end = datetime.fromtimestamp(int(end_date))
```

Comment:
function signature already has type, I think its not necessary to use `int()` in this case